test/near_provider.test.js

```diff
@@ -8,10 +8,10 @@ const web3 = require('web3');
 const bn = web3.utils.BN
 const nearlib = require('near-api-js');
 const utils = require('../src/utils');
-const { NearProvider } = require('../src/index');
+const { NearProvider, nearWeb3Extensions } = require('../src/index');
 
 // TODO: update nearEvmFile frequently when near_evm work is being done
-const nearEvmFile = './artifacts/near_evm.wasm';
+const nearEvmFile = './artifacts/near_evm0.wasm';
```

Comment:
Is this a new wasm file? Does it need to be pushed up?

Reply:
oh shoot, thought I had taken this out, thanks. (no I was just logging)